wordpress-mobile/WordPress-iOS
# Reader Post Details Comments: enable Follow Conversation

Ref: #17632 

This enables Follow Conversation from Comments snippet for release.

To test:
- Go to Reader > post > details.
- Verify the Follow button appears in the Comments header and functions as expected.

Detailed testing steps can be found [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/17739).

| <img width="472" alt="not_following" src="https://user-images.githubusercontent.com/1816888/149223836-cd13a749-0b41-49f9-bd83-e4cf299958cb.png"> | <img width="472" alt="following" src="https://user-images.githubusercontent.com/1816888/149223842-cf0f5729-5b90-4d94-81e8-a8163352d074.png"> |
|--------|-------|

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
